CHANGELOG.md

```diff
@@ -5,10 +5,24 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [0.7.0] - 2023-03-10
+### Added
+- Added support for Conjur's OIDC and LDAP authenticators
+  [cyberark/summon-conjur#108](https://github.com/cyberark/summon-conjur/pull/108)
+
+### Changed
+- Updated Golang to 1.19
+  [cyberark/summon-conjur#108](https://github.com/cyberark/summon-conjur/pull/108)
+
 ### Security
 - Update golang.org/x/sys to v0.1.0 for CVE-2022-29526 (not vulnerable)
   [cyberark/summon-conjur#110](https://github.com/cyberark/summon-conjur/pull/110)
 
+### Removed
+- Removed support for Conjur v4
```

Comment:
In doing this we're moving to test summon-conjur against Conjur OSS, with the understanding that the interface for enterprise is identical ?

Reply:
Yes. @jvanderhoof can you confirm this is a safe assumption?

Reply:
It's true! 😄. Enterprise adds health and info, but nothing to the primary API.